addon/components/power-select-infinity.js

```diff
@@ -42,23 +42,23 @@ export default Component.extend({
     return classes.join(' ');
   }),
 
-  actions: {
-      handleFocus(select) {
-          if (!isEmpty(select.options)) {
-              scheduleOnce('actions', select, select.actions.search);
-              scheduleOnce('actions', select, select.actions.open);
-          }
-      },
-      onKeyDown(select, e) {
-          let action = get(this, 'onkeydown');
+    actions: {
+        async handleFocus(select) {
+            await select.actions.search();
```

Comment:
@bakerac4 does this mean on focus a new search request will be made with the same/existing term, waiting for that to complete before re-opening the results dropdown? would it be possible to just immediately reopen the menu with the previously fetched results displayed?

Reply:
@billdami the search is performed, which means if the search term is blank it will resolve immediately, and if there are options will then open.